modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/ComponentMessageProcessor.java

```diff
@@ -200,6 +202,7 @@
   protected CompletableComponentExecutor componentExecutor;
   protected ReturnDelegate returnDelegate;
   protected PolicyManager policyManager;
+  protected ClassLoader classLoader;
```

Comment:
can we add a more declarative name for this classloader (to avoid aliasing too). For example classloaderToNestedChain.

Reply:
Done.